shortcircuit-api/src/main/java/com/powsybl/shortcircuit/VoltageRangeData.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.shortcircuit;
+
+import com.powsybl.commons.PowsyblException;
+import org.apache.commons.lang3.Range;
+
+import java.util.List;
+
+/**
+ * A class that stores coefficients to be applied to every nominal voltage in a range. This is used to define the configured initial voltage profile
+ * for short circuit calculation.
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
+ */
+public class VoltageRangeData {
+    final Range<Double> voltageRange;
```

Comment:
This would be come just range as it is an attribute in the context of a `VoltageRange`